R/collectors.R

```diff
@@ -138,8 +138,23 @@ col_skip <- function() {
 #' @family parsers
 #' @export
 #' @examples
-#' parse_number("$1000")
+#' ## These all return 1000
+#' parse_number("$1,000") ## leading $ and grouping character , ignored
+#' parse_number("£1000")  ## leading non-numeric £ ignored (locale doesn't matter)
```

Comment:
I would avoid adding a challenging character like this.

@jimhester what do you think?

Reply:
Yeah I think that is right, it actually is a great example, but unfortunately non-ASCII characters often cause us problems in examples on CRAN

Reply:
@cdhowe I think we have to remove this example unfortunately